liquibase-standard/src/main/java/liquibase/change/custom/CustomChangeWrapper.java

```diff
@@ -159,19 +159,17 @@ public Warnings warn(Database database) {
     @Override
     public SqlStatement[] generateStatements(Database database) {
         SqlStatement[] statements = null;
-        if (shouldExecuteChange(database)) {
```

Comment:
@wwillard7800 if I'm right this shouldExecuteChange was added to fix a problem when we were executing this change from mdc logging or something like that. Is it ok to remove it?

Reply:
Yes, I found that it was okay to remove it.  I think it was unnecessary to add it when I did.